generic_tracers/generic_COBALT_diagyaml2register_diag_body.py

```diff
@@ -0,0 +1,110 @@
+"""
+File: 
+    generic_COBALT_diagyaml2register_diag_body.py
+Author: 
+    Kelly Kearney
+Date: 
+    2024/03/27
+Description: 
+    This script builds the Fortran source code that makes up the primary body of the 
+    generic_COBALT_register_diag subroutine.  This routine allows for easy maintenance of the 
+    highly-repetitive and lengthy code.  The resulting generic_COBALT_register_diag_body.f90 is 
+    added to the primary generic_COBALT.f90 module via an include statement, and can be quickly 
+    regenerated when new diagnostics are added or the specific commands in question need to be 
+    modified.
+Usage:
+    python generic_COBALT_diagyaml2register_diag_body.py formatopt
+
+    formatopt is a string indicating the code format to use:
+        original:   The format in the original generic_COBALT.  A (temporary) vardesc-formatted 
+                    variable is created for each diag, then register_diag_field is called using 
+                    fields from that, using pass-by-position syntax for the first 6 inputs and 
+                    name/value for any additional (default if script called w/o input)
+        novardesc:  This format eliminates the creation of an intermediate vardesc variable 
+"""
+
+import yaml
+import sys
+
+# Parse and check input
+
+if len(sys.argv) < 1:
```

Comment:
Should this be `if len(sys.argv) < 2:` if we want to keep `original` as the default when users do not provide an input argument?